script/Ciel/Day 12/12_01_CIEL12_5.txt

```diff
@@ -551,27 +595,27 @@ There's no need to look at her lines of death.
 [sha:5174d821e58ad64826de843f3b94af0c181f5fdf]{
 -- Page 247, Offset 31602.
 -- それだと、本当に。
-                                    If I do that―――
+                                     If I do that―――
 }
 [sha:38fdfe7d58a9903785b94ce621a59957b082013a]{
 -- Page 247, Offset 31603.
 -- 躊躇う意味がない。なぜならおまえは
-There's no need for hesitation. After all, you've―――
+There's no need for hesitation. After all...
 }
 [sha:5095383b2c4b1cfabcadd5bdca42ae819b13bf84]{
 -- Page 247, Offset 31604.
 -- 俺はこのまま、
-                               If I don't stop―――
+                                  If I don't stop―――
 }
 [sha:b7d5fb426bb9ddec473f2917dcfc4e8396b8cf20]{
 -- Page 247, Offset 31605.
 -- もう幾度となく、白い首筋を切断してきただろう？
-already cut through these pale necks so many times, haven't you?
+You've already sliced through plenty of pale necks, haven't you?
 }
 [sha:23eaaaff54d5e18ae6edbc82592f0693ce1844fa]{
 -- Page 247, Offset 31606.
 -- 畜生以下の生き物になる。
-                      I'll become a being even worse than a beast.
+           I'll become a being even worse than a beast.
```

Comment:
Any of these spark something?
> I will debase my very existence.
My hands will be permanently stained.
Then I will be beyond saving.

Reply:
Hmmm, first one is a bit too mild
Second is interesting, but it's not self-deprecating enough
Same for the third.
The most important factor I think is the extreme levels of self-deprecation.

This is what I could think of, but I don't love it. Though highly preferrable to the current.
>I really%{i}will%{/i} become the most despicable creature there is.

Another thing is that the 本当に at the start got lost somewhere, so it might as well come back here. I think it fits in pretty well.